communication/website-guidelines.md

```diff
@@ -0,0 +1,33 @@
+# Website guidelines
+
+## General content
+
+The SIG-Docs [website subproject] maintains the kubernetes.io website,
+including site content, documentation, and generated reference documentation.
+SIG-Docs maintains guides for website contributors, including a [content guide]
+detailing the types of content allowed, and a [style guide].
+
+[Contribute to Kubernetes docs]
+
+## Blog 
+
+The [Kubernetes Blog] is a subproject of SIG-Docs and operated by the [blog team].
+
+[Submit a blog post]
+
+## Community statements
+
+Prominent notices on Kubernetes web sites (such as site-wide or front-page banners)
```

Comment:
nit: I assume that these guidelines apply to _any_ Kubernetes community-related website (not only `kubernetes.io`).
nit: s/web site/website/
/lgtm